File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Gmmktime.java
package com.onevgo.functions;

import java.time.LocalDateTime;
import java.time.ZoneOffset;
import java.util.Date;

public class Gmmktime {
    public static long gmmktime(int hour, int minute, int second, int month, int day, int year) {
        return LocalDateTime.of(year, month, day, hour, minute, second).toInstant(ZoneOffset.UTC).getEpochSecond();
    }

    public static void main(String[] args) {
        System.out.println(new Date(gmmktime(0, 0, 0, 7, 1, 2000) * 1000));
    }
}

File: 01_Language/01_Functions/python/gmmktime.py
# coding: utf-8

import calendar


def gmmktime(hour, minute, second, month, day, year):
    return calendar.timegm((year, month, day, hour, minute, second))


if __name__ == '__main__':
    print(gmmktime(0, 0, 0, 7, 1, 2000))
